src/Summary/SummaryForm.js

```diff
@@ -1,105 +1,189 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { FormattedMessage } from 'react-intl';
 import PropTypes from 'prop-types';
-import { Field, FieldArray } from 'redux-form';
 import {
-  Row,
-  Col,
+  Field,
+  FieldArray,
+} from 'redux-form';
+
+import {
   Button,
-  TextField,
-  TextArea,
-  Select,
   Checkbox,
+  Col,
+  Row,
+  Select,
+  TextArea,
+  TextField,
 } from '@folio/stripes/components';
+
 import { Required } from '../Utils/Validate';
 import css from './SummaryForm.css';
 
-class SummaryForm extends React.Component {
+const ORGANIZATION_STATUS = {
```

Comment:
IMHO, it's better to pull up into `src/common/constants/organization` or something

Reply:
Done